zulip/zulip/__init__.py

```diff
@@ -1488,6 +1488,13 @@ def get_stream_topics(self, stream_id: int) -> Dict[str, Any]:
         """
         return self.call_endpoint(url=f"users/me/{stream_id}/topics", method="GET")
 
+    def get_stream_email_address(self, stream_id: int) -> Dict[str, Any]:
+        """
+        Example usage: client.get_stream_email_address(stream_id)
+        """
+        url = f"streams/{stream_id}/email_address"
+        return self.call_endpoint(url=url, method="GET")
```

Comment:
This looks functional enough; have you manually tested this?

When contributing to a codebase it's useful to look at nearby code to match the style. My thoughts would be:
- the example doesn't use literals, which others nearby do?
- some nearby examples use a repl-like format, and show an idea of an expected successful response
- most nearby functions inline the url

A question for someone else: does this need backwards-compatibility for servers that don't have this endpoint?

Reply:
Thanks for reviewing. I've updated the PR with the given feedback. 

I have tested this manually by putting it into the ZT model. Are there alternative methods for manual testing that I might not be familiar with?

Reply:
Within ZT should work fine; I didn't know what kind of testing you'd performed :)

I just tested this manually in a REPL environment:
```python3
import zulip
c = zulip.client(config_file="...")
s_id = c.get_stream_id("zulip-terminal")
print(c.get_stream_email_address(s_id))
```
This produces an output which matches what I'd expect for the example in the code, and matches the default output by the web client for me.

Reply:
Oh that makes sense, thanks for letting me know :)